nemo/collections/nlp/models/language_modeling/megatron_lm_encoder_decoder_model.py

```diff
@@ -442,7 +442,10 @@ def fwd_output_only_func(batch, model):
                 decoder_attn_mask,  # dec_attn_mask
                 None,  # token_type_ids
                 None,  # labels
-                enc_input,  # enc_hidden_states
+                None,  # enc_hidden_states
```

Comment:
This bit is being refactored in this PR - https://github.com/NVIDIA/NeMo/pull/4210. Do we have to add all these additional args to make it work?

Reply:
No. I just temporarily modify the args so that `enc_input` can be utilized in the `decode` function. I see the changes in that PR. If it gets merged first, I can change my codes based on it.